environment.yml

```diff
@@ -3,7 +3,8 @@ channels:
   - conda-forge
 dependencies:
   - jupyterlab
-  - napari
+  - matplotlib
+  - napari[all]
```

Comment:
is this a thing in conda?

Reply:
no, I mean, I thought optional dependencies and `[]` notation didn't exist in conda, that's why there's matplotlib-base etc?

Also, maybe just switch to mamba? mamba is just lightning for environment creation...

Reply:
Personally that's what I do but I'm not sure it is a thing in all OSs. This environment.yml file does work with conda though, I tested again on my own machine and see no issues with both

`mamba env create -f environment.yml`

and 

`conda env create -f environment.yml`

Reply:
Actually I'm an idiot. I don't know what is happening to `[all]` here but I think it is just silently ignored — I'm a maintainer on the conda-forge recipe and I'm 99% sure that we don't provide an "all" package. ie I think you should just change this to "napari" by itself and you will see the same results. CC @jaimergp in case I missed something 😅